lib/ansible/executor/module_common.py

```diff
@@ -1186,9 +1185,15 @@ def _find_module_utils(module_name, b_module_data, module_path, module_args, tas
                     # so that no one looking for the file reads a partially
                     # written file)
                     if not os.path.exists(lookup_path):
-                        # Note -- if we have a global function to setup, that would
-                        # be a better place to run this
-                        os.makedirs(lookup_path)
+                        try:
+                            # Note -- if we have a global function to setup, that would
+                            # be a better place to run this
+                            os.makedirs(lookup_path)
+                        except OSError:
```

Comment:
@samdoran Was there a reason for not just catching `FileExistsError` here? The `OSError` superclass feels fairly broad since we only care about the case where the directory already exists (due to the race). I have slight concerns that this will mask a different kind of `OSError` and try to continue on in an unknown state when it should have failed out.

Reply:
`FileExistsError` is Python 3 only.

Reply:
We could do some additional inspection of the error, like `os.makedirs` used to do.